packages/contracts-bedrock/contracts/L2/CrossDomainOwnable3.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Predeploys } from "../libraries/Predeploys.sol";
+import { L2CrossDomainMessenger } from "./L2CrossDomainMessenger.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title CrossDomainOwnable2
+ * @notice This contract extends the OpenZeppelin `Ownable` contract for L2 contracts to be owned
+ *         by contracts on L1. Note that this contract is meant to be used with systems that use
+ *         the CrossDomainMessenger system. It will not work if the OptimismPortal is used
+ *         directly.
+ */
+abstract contract CrossDomainOwnable3 is Ownable {
+    /**
+     * @notice If true, the contract uses the cross domain _checkOwner function override. If false
+     *         it uses the standard Ownable _checkOwner function.
+     */
+    bool internal isLocal = true;
```

Comment:
I think that this should be public and we should test it being changed after its changed